README.md

```diff
@@ -6,9 +6,31 @@ We want to raise awareness of environmental sustainability as a key element of o
 
 The [Environmental Sustainability](https://github.com/cncf/env-sustainability-wg/charter.md) WG Charter further outlines the scope of our group activities as well as the goals and intended deliverables.
 
+## Governance
+
+The Environmental Sustainability [charter](charter.md) outlines the scope of our group activities, as part of our [governance process](/governance/README.md) which details how this group works together.
+
 ## Meetings
 
-* The working group meets every Wednesday at 5pm CEST/GMT+2 on [Zoom](https://zoom.us/my/cncfwgenvsustainability)
+* The working group meets every Wednesday at 5pm CEST/GMT+2 or 8am PST
+  * Meeting Link: https://zoom.us/my/cncfwgenvsustainability
+  * Password: 77777
 * Meeting notes & agenda can be found [here](https://docs.google.com/document/d/1TkmMyXJABC66NfYmivnh7z8Y_vpq9f9foaOuDVQS_Lo/edit#)
 * Public discussion runs on https://lists.cncf.io/g/cncf-wg-env-sustainability/ via mailto:cncf-wg-env-sustainability@lists.cncf.io
 * Reach us at [#wg-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6) slack channel on slack.cncf.io
+
+## Members
+
+> The WG is currently in its initialization phase, members need to be elected, projects needs to be organized and roles assigned
+
+### Environmental Sustainability Chairs
+
+* Max Körbächer ([@mkorbi](https://github.com/mkorbi)), Liquid Reply [Chair term: 01.06.2022 - 01.06.2024]
+* Leonard Pahlke ([@leonardpahlke](https://github.com/leonardpahlke)), Liquid Reply [Chair term: 01.06.2022 - 01.06.2024]
```

Comment:
> AFAIK there are no current co-chairs. Why are Max and Leonard added to this PR as such?

Like with other proposed groups, the initiator gets this responsibility. I know not every WG has chairs, complex ones like the CNF do so. 

> Max thank you for sending that email. However, I think we should extend the proposed nomination period there (I will note that on the email list). For now, let's at least fix this PR and connect with CNCF on it, as the process so far is a bit atypical. We'll all get it together as a group 😄

As answered already in the mailing list, that is not correct. At the moment there is no nomination going on. It was proposed to postpone this for some weeks because the discussed/nominated people in the meeting are basically unknown, haven't contributed yet on any of the points and were a little too fast forward.